frontend/packages/ux-editor/src/types/Expressions.ts

```diff
@@ -0,0 +1,35 @@
+
+// Could we instead collect all properties from the specific component that has the type boolean?
```

Comment:
Maybe at some point, but not all boolean properties on all components actually support expressions. 
That being said, once we get better schema-support for all components, we will know exactly which properties are set up with expression support, so at a later stage we could do what you are suggesting!